GVFS/GVFS.Upgrader/UpgradeOrchestrator.cs

```diff
@@ -319,21 +282,21 @@ private bool TryCheckIfUpgradeAvailable(out Version newestVersion, out string co
 
                 if (newestVersion == null)
                 {
-                    consoleError = "No upgrades available in ring: " + this.upgrader.Ring;
+                    consoleError = "Upgrade is not available.";
                     this.tracer.RelatedInfo("No new upgrade releases available");
                     return false;
                 }
 
                 activity.RelatedInfo("Successfully checked for new release. {0}", newestVersion);
             }
-            
+
             return true;
         }
 
         private bool TryDownloadUpgrade(Version version, out string consoleError)
         {
             using (ITracer activity = this.tracer.StartActivity(
-                $"{nameof(this.TryDownloadUpgrade)}({version.ToString()})", 
+                $"{nameof(this.TryDownloadUpgrade)}({version.ToString()})",

```

Comment:
Added it to metadata as well. Keeping it in the activityName is useful, it shows the version being downloaded in the logs even when there is no error. 

Like this -
```
[2018-12-19 11:46:11 -05:00] TryDownloadUpgrade(1.0.18297.1) (Start)
[2018-12-19 11:46:15 -05:00] Information {"Message":"Successfully downloaded version: 1.0.18297.1"}
[2018-12-19 11:46:15 -05:00] TryDownloadUpgrade(1.0.18297.1) (Stop) {"DurationMs":3300}
```

Reply:
@alameenshah sorry I should have been more clear in my suggestion.  `StartActivity` itself can take metadata, what does the logging look like if you pass the version in there (rather than as part of the activity name)?

Reply:
oh ok. Here is how it looks -

```
[2018-12-20 12:40:48 -05:00] TryDownloadUpgrade(1.0.18297.1) (Start) {"Upgrade Step":"TryDownloadUpgrade","Version":"1.0.18297.1"}
[2018-12-20 12:40:54 -05:00] Information {"Message":"Successfully downloaded version: 1.0.18297.1"}
[2018-12-20 12:40:54 -05:00] TryDownloadUpgrade(1.0.18297.1) (Stop) {"DurationMs":6534}
```

Now version info gets logged twice. And your first comment to remove metadata from activityName makes sense, will remove it.